src/libs/nav.ts

```diff
@@ -83,15 +121,15 @@ export const getCurrentRoute = () => {
 };
 
 export const useRoute = () => {
-  const location = useContext(locationContext);
+  const location: any = useContext(locationContext);
```

Comment:
```suggestion
  const location = useContext(locationContext);
```

Do we need the explicit `any`? It looks like the type of this would still be `any` without it (because `locationContext` is typed `Context<any>`).

Reply:
once we give nav.ts more Typescript love in general, things should click in better.  But i was running into type flow problems where Context<any> was not enough with the current state of related types and usage.